### PR TITLE
Hide recipe selection from the enroll discovered application wizard

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2291301

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
plz add screenshot and fix build tests...

**Review by alfonsomthd — COMMENTED** [review]
/lgtm

**Review by alfonsomthd — COMMENTED** [review]
/retest

**Comment by alfonsomthd** [comment]
@TimothyAsirJeyasing A fix is needed fot this test:
`
FAIL packages/mco/components/discovered-application-wizard/enroll-discovered-application.spec.tsx (13.556 s)
  ● Test configure step › Configure form test
`

File: packages/mco/components/discovered-application-wizard/enroll-discovered-application.spec.tsx [hunk 1]
```diff
@@ -288,20 +288,21 @@ const moveToStep = async (step: number) => {
...
     // Next wizard step
     fireEvent.click(screen.getByText('Next'));
   }
 
   if (step > 3) {
     // Select policy
-    fireEvent.click(screen.getByText('Select a policy'));
-    fireEvent.click(screen.getByText('mock-policy-1'));
+    // fireEvent.click(screen.getByText('Select a policy'));
+    // fireEvent.click(screen.getByText('mock-policy-1'));
```
Comment (SanjalKatiyar) [inline packages/mco/components/discovered-application-wizard/enroll-discovered-application.spec.tsx:305]: this is not Recipe test, why commenting this ?

Comment (TimothyAsirJeyasing) [reply packages/mco/components/discovered-application-wizard/enroll-discovered-application.spec.tsx:305]: Sorry, did't noticed, yes its not the Recipe test, will remove the comment.

**Comment by SanjalKatiyar** [comment]
/lgtm cancel
cc @GowthamShanmugam for LGTM label

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**